integration-test/src/main/java/com/sequenceiq/it/cloudbreak/cloud/v4/aws/AwsCloudProvider.java

```diff
@@ -492,6 +480,19 @@ public String getStorageOptimizedInstanceType() {
         return awsProperties.getStorageOptimizedInstance().getType();
     }
 
+    @Override
+    public void assertServiceEndpoint(EnvironmentTestDto environmentTestDto) {
```

Comment:
what is the reason to add an assertion to the CloudProvider? This class provide and prepare the dto and it hasn't an assertion responsibility.

I prefer to create a provider-specific assertion system instead of reusing another but different responsibility system.

Reply:
CloudProviderAssertion was introduced, analogous to CloudProvider.